### ISSUE TITLE
How to deal with encounter (visit) information in BICEPS

### ISSUE THREAD

**Issue by PeterKranich** [IHE/sdpi-fhir#331]
## Section Number

This relates to @MartinKasp comment in #288. B.3.4.4.4 PV1-19 Visit Number

## Priority

- High: Important issue where there is major issue to be resolved. Requires discussion and debate.

## Issue

The issue surfaced in the IHE HL7 V2 gateway section but is a general issue about how to deal with patient encounters (alias visits).

## Proposed Change

@MartinKasp recommended to use the MDIB WorkflowContext for this concept, but from the BICEPS definition it is not clear how to use it in real clinical workflows. FHIR has a good decription for the Encounter conecpt (http://hl7.org/fhir/administration-module.html). The WorkflowContext has a WorkflowDetail element which has elements for patient, AssignedLocation, VisitNumber, etc. but how does this relate to the existing PatientContextState and LocationContextState?
Use case need to be defined and mapped to the SDPi requirements for the WorkflowContext.

**Comment by AnnaFeiler** [comment]
Hi Peter, I agree that this is a very important discussion. But since we have postponed developing a concept how to build patient/location based ensembles from patient and location contexts for the different use cases as well as how to get patient data distributed at a workplace (share at point of care vs. import fropm hospital information systems), it might be difficult to solve right now. All of those topics are at the top of my (currently not aligned) to do list for version 1.1. Could we solve this as part of this bigger discussion once SDPi 1.0 is done?

**Comment by AnnaFeiler** [comment]
Added a related issue to have all information in one place.

**Comment by riechkathrin** [comment]
This also has an effect on: #258
